src/client/components/App/index.js

```diff
@@ -57,7 +57,7 @@ const App = () => {
         />
       )}
       {currentTab === constants.TABS.COURSE_RESERVES && (
-        <CourseReserves url="" />
+        <CourseReserves context={platformContext} />
```

Comment:
Why not get this automatically from lti in the CourseReserves rather than sending as param?

Reply:
@finnzink I just thought it would be more efficient to pass it in since it is already being retrieved from lti in App/index.js, no need to do that twice